docs/getting_started/quickstart.md

```diff
@@ -33,22 +33,11 @@ The Triton Inference Server is available as [buildable source
   use the pre-built Docker image available from the [NVIDIA GPU
   Cloud (NGC)](https://ngc.nvidia.com).
 
-## Install Triton Docker Image
+Launching and maintaining Triton Inference Server revolves around the use of building model repositories. This tutorial will cover:
 
-Before you can use the Triton Docker image you must install
-[Docker](https://docs.docker.com/engine/install). If you plan on using
-a GPU for inference you must also install the [NVIDIA Container
-Toolkit](https://github.com/NVIDIA/nvidia-docker). DGX users should
-follow [Preparing to use NVIDIA
-Containers](http://docs.nvidia.com/deeplearning/dgx/preparing-containers/index.html).
-
-Pull the image using the following command.
-
-```
-$ docker pull nvcr.io/nvidia/tritonserver:<xx.yy>-py3
-```
-
-Where \<xx.yy\> is the version of Triton that you want to pull.
+* Creating a model Repository
```

Comment:
Capital "Model Repository" and add link to model repository doc

Reply:
Made the capitalization but this bullet is for the section below. Linking model repo doc will break the flow